src/app/beer_garden/api/http/schemas/v1/user.py

```diff
@@ -73,3 +82,17 @@ def dump_obj(self, data, many=False, **kwargs):
         for item in data:
             for assignment in item.get("role_assignments", []):
                 assignment["role_name"] = assignment.pop("role")["name"]
+
+
+class UserSchema(BrewtilsUserSchema):
+    sync_status = fields.Dict(dump_only=True)
+
+    @pre_dump
+    def get_sync_status(self, user):
+        from beer_garden.user import user_sync_status
```

Comment:
The way the code is organized makes it increasingly difficult to avoid circular dependencies.  Specifically the relationship between the models.py and the user.py that houses all of the helpers for interacting with Users.